Orange/widgets/cpewidget.py

```diff
@@ -126,7 +147,10 @@ def property_changed(self, name):
 
             setattr(self.node, name, value)
             # Synchronize property changes back to the GUI.
-            super().__setattr__(name, getattr(self.node, name))
+            if setNoneUIValue:
```

Comment:
If you called the same method here, that one would check if value is None, which is true whenever setNoneUIValue would have been true, so you can remove the whole setNoneUIValue thing, which makes this much simpler (only one thing to care about, namely content/value).